lib/winrm-elevated/scripts/elevated_shell.ps1

```diff
@@ -88,7 +88,7 @@ try {
 
 function SlurpOutput($file, $cur_line, $out_type) {
   if (Test-Path $file) {
-    get-content $file | Select-Object -skip $cur_line | ForEach-Object {
+    get-content -Encoding Oem $file | Select-Object -skip $cur_line | ForEach-Object {
```

Comment:
Unfortunately `Oem` is not supported in powershell 2. Even though that is ancient, it would be nice to continue support while it is feasible. Changing `SlurpOutput` to:
```
  if (Test-Path $file) {
    $fs = New-Object -TypeName System.IO.FileStream -ArgumentList @(
      $file,
      [system.io.filemode]::Open,
      [System.io.FileAccess]::Read,
      [System.IO.FileShare]::ReadWrite
    )
    try {
      $enc = [System.Text.Encoding]::GetEncoding($Host.CurrentCulture.TextInfo.OEMCodePage)
      $bytes = [System.Byte[]]::CreateInstance([System.Byte], $fs.Length)
      if ($fs.Read($bytes, 0, $fs.Length) -gt 0) {
        $text = $enc.GetString($bytes)
        $text.TrimEnd("`n").TrimEnd("`r").Split(@("`r`n", "`n"), [StringSplitOptions]::None) | Select-Object -skip $cur_line | ForEach-Object {
          $cur_line += 1
          if ($out_type -eq 'err') {
            $host.ui.WriteErrorLine("$_")
          } else {
            $host.ui.WriteLine("$_")
          }
        }
      }
    }
    finally { $fs.Close() }
  }
  return $cur_line
```
will work in powershell 2.

Reply:
@mwrock At this point it could be better to solve #44 so that we won’t need any explicit conversion at all. I haven’t had any luck with PowerShell’s redirections and Write-Host, but if you’re familiar with older versions of PowerShell maybe you’ll know a way and we won’t have to resort to a manual slurp.

Reply:
After some thought, and since my proposal for Unicode breaks special characters for ipconfig, maybe it’s better to support the ANSI/OEM encoding first, which should cover most use cases without black magic.

I’ve applied your proposed changes. Thanks for the review!